lib/id3c/cli/command/etl/fhir.py

```diff
@@ -553,13 +553,19 @@ def related_specimens(encounter: Encounter, resources: Dict[str, List[DomainReso
         else:
             assert False, "logic bug"
 
+        sample_details = {}
+        if specimen.note:
+            sample_details['note'] = specimen.note[0].text
+
+        additional_details = { **specimen.type.as_json(), **sample_details}
```

Comment:
This would leave us with both the `note` and `notes` key; the latter comes from the manifest.

I'd like to avoid this if we can…

Reply:
I forgot about the `notes` key. 

Should it be named `status_code`?

Reply:
Stick with the `able_to_test` key?

Reply:
Hmm. I'd also like to avoid encoding more study-specific handling of the data into this FHIR ingest, which is supposed to be trending towards generic.

I do think the current handling of note in this PR is good and fine. It's only the interaction with our manifest ETL that's poor. Perhaps we should ingest manifest notes as a different key altogether and do a bulk-update, for example?

Reply:
I added a [Trello card](https://trello.com/c/seCBMUxV/485-store-specimen-manifest-details-under-manifest-key-in-sampledetails) to move the specimen manifest details under a new key in `sample.details`. 

OK to keep the `Specimen.note` annotation under `sample.details->>'note'` (as I've done here)?

Reply:
Yes!